tests/Unit/Domain/Creators/Test_AlignedLattice.cpp

```diff
@@ -65,5 +65,49 @@ SPECTRE_TEST_CASE("Unit.Domain.Creators.AlignedLattice", "[Domain][Unit]") {
       dynamic_cast<const creators::AlignedLattice<3, Frame::Inertial>*>(
           domain_creator_3d.get());
   test_aligned_blocks(*aligned_blocks_creator_3d);
+
+  const auto rubiks_cube_with_hole_domain =
```

Comment:
Do you call this rubiks cube, because it is supposed to be 3x3x3?

Will we run into trademark issues, when we use this name?

Reply:
I'd hope not, but I'll make the change to `cubical_shell` regardless :+1: